wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewControllerViewModel.swift

```diff
@@ -50,7 +50,7 @@ protocol ProfileViewControllerViewModeling {
     func sendConnectionRequest()
     func acceptConnectionRequest()
     func ignoreConnectionRequest()
-    // TODO: Make it async https://wearezeta.atlassian.net/browse/WPB-9028
```

Comment:
is it part of this PR on purpose ?

Reply:
When I pulled in the base branch I pulled in several swift lint violations which needed to be corrected, including this.